annotations/src/main/java/org/robolectric/annotation/ClassName.java

```diff
@@ -5,7 +5,25 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
-/** Parameters with types that can't be resolved at compile time may be annotated @ClassName. */
+/**
+ * Parameters with types that can't be resolved at compile time may be annotated @ClassName.
+ *
+ * <p>Use this annotation when creating shadow method for newly android SDK which could have problem
+ * resolving the parameter type in lower android SDK.
+ *
+ * <p>Effectively replace the usage of looseSignatures.
```

Comment:
I would remove `Effectively replace the usage of looseSignatures.` as we will be deleting looseSignatures, hopefully soon.

Reply:
done